my-first-website/index.html

```diff
@@ -1,9 +1,37 @@
-ABOUT
-Hi, My name is Randy. I aspire to become a software engineer, and I am proud to participate in the Code The Dream Bootcamp that aims to promote equity in education and equal opportunity to their students of all backgrounds to further develop valuable professional abilities while inspiring students to succeed.
-    
-Experience
-Code the Dream Bootcamp
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Aspiring Software Engineer</title>
+    <meta charset="UTF-8">
+    <meta name="description" content="About Me">
+    <meta name="keywords" content="HTML, CSS, JavaScript">
+    <meta name="author" content="Randy's About Webpage">
+</head>
+<body>
+    <section>
+        <h1>ABOUT</h1>
 
-Connect
-<a href="#">Email</a>
-<a href="#">Github</a>
+        <img src="/Users/randy/intro-to-programming-asteroid/media/Code programmer.jpeg" alt="Code Programmer Picture">
+
+        <p>Hi, My name is Randy. I aspire to become a software engineer, and I am proud to participate in the <strong>Code The Dream</strong> Bootcamp that aims to promote equity in education and equal opportunity to their students of all backgrounds to further develop valuable professional abilities while inspiring students to succeed.</p>
+    </section>
+
+    <section>
+        <h2>Experience</h2>
+        <nav>
+            <ul>
+                <li>Code the Dream Bootcamp</li>
+            </ul>
```

Comment:
did you know that an unordered list starts with the <ul> tag. Each list item starts with the <li> tag and the list items will be marked with bullets (small black circles) by default. An ordered list starts with the <ol> tag. Each list item starts with the <li> tag.

The list items will be marked with numbers by default: